src/utils.tsx

```diff
@@ -71,17 +71,10 @@ export const getUserClass = (userClass: UserClass) => {
 const addLeadingZero = (number: number) => (number < 10 ? '0' + number : number);
 
 export const formatDate = (date: Date) => {
-  return (
-    getDay(date.getDay()) +
-    ' ' +
-    date.getDate() +
-    ' ' +
-    getMonth(date.getMonth()) +
-    ' - kl. ' +
-    addLeadingZero(date.getHours()) +
-    ':' +
-    addLeadingZero(date.getMinutes())
-  );
+  const isEarlierYear = date.getFullYear() < new Date().getFullYear();
```

Comment:
```suggestion
  const isDifferentYear = date.getFullYear() !== new Date().getFullYear();
```

Kanskje vise år også hvis det er et fremtidig år, ikke relevant veldig ofte men like greit å ha

Reply:
ja word